user_guide_src/source/libraries/sessions.rst

```diff
@@ -346,12 +346,13 @@ Destroying a Session
 ====================
 
 To clear the current session (for example, during a logout), you may
-simply use either PHP's `session_destroy() <https://www.php.net/session_destroy>`_
-function, or the library's ``destroy()`` method. Both will work in exactly the
-same way:
+simply use the library's ``destroy()`` method:
 
 .. literalinclude:: sessions/037.php
 
+This method will work in exactly the same way as PHP's
+`session_destroy() <https://www.php.net/session_destroy>`_ function.
```

Comment:
The method just calls `session_destroy()`, so this is correct.

But when using the Session library, a cookie that removes the session cookie will be issued
if you call `Session::destory()` or `session_destroy()`.
This behavior is different from PHP's `session_start()` and `session_destroy()`.

I don't know why the cookie is issued.